examples/options/include/traccc/options/particle_gen_options.hpp

```diff
@@ -0,0 +1,72 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/options/options.hpp"
+
+// Boost
+#include <boost/program_options.hpp>
+
+namespace traccc {
+
+namespace po = boost::program_options;
+
+template <typename scalar_t>
+struct particle_gen_options {
+    unsigned int gen_nparticles{1};
+    std::array<scalar_t, 3> vertex{0., 0., 0.};
+    std::array<scalar_t, 3> vertex_stddev{0., 0., 0.};
+    std::array<scalar_t, 2> mom_range{1., 1.};
+    std::array<scalar_t, 2> phi_range{0., 0.};
+    std::array<scalar_t, 2> theta_range{0., 0.};
```

Comment:
Why didn't you use the `traccc::Reals` type for these as well? :confused: If we only use that type behind the scenes, we may as well live without that definition... :thinking:

Reply:
Right. I think I used `std::array` to match them with the input types of `simulator` - but using `Reals` should be OK as well